hkmc2/shared/src/test/mlscript/backlog/Lifter.mls

```diff
@@ -2,9 +2,28 @@
 :lift
 :todo
 
+
+// The following are problems with lifting functions inside other functions.
+
+
+// Lifting functions with spread arguments is broken.
+:expect [1]
+fun f(x) =
+  fun g(...rest) = 
+    print(x)
+    rest
+  let a = g
+  a(1)
+f(2)
```

Comment:
That test case is already in `FunInFun.mls`.

We _really_ don't want to duplicate test cases! De-duplication of test cases is horribly laborious and no one will want to do it, so we'll just end up with many redundant tests.

Reply:
Oh? All the tests in the backlog are copied directly from the main tests. I thought the point is that the backlog tests would be removed as they are fixed.

Reply:
The backlog is just for broken stuff to fix later. Either you add the broken thing to the backlog and when it's fixed we'll move it where it belongs (you can specify that in a comment if needed). Or you leave the test here it belongs with a `:todo` or `:fixme` and ideally also update the issue so we don't lose track of it.